zenoh/src/net/runtime/orchestrator.rs

```diff
@@ -1024,7 +1024,7 @@ impl Runtime {
         }
     }
 
-    pub async fn connect_peer(&self, zid: &ZenohIdProto, locators: &[Locator]) {
+    pub async fn connect_peer(&self, zid: &ZenohIdProto, locators: &[Locator]) -> bool {
```

Comment:
Correct me if I'm wrong. `Runtime::connect` returns true if it has opened a new Transport with `zid`. This seems to do the opposite.

Reply:
If I understand correctly, `connect_peer` only returns `false` if `Runtime::connect` returns `false`, and `Runtime::connect` returns `true` if it succeeds in opening a new transport with `zid`.
The only confusing bit to me is that if a connection is pending with `zid`, then both functions will return `false`.

Reply:
Nevermind, it returns true if a Transport is opened with `zid`.